### PR TITLE
update restic docs for v0.9.0 GA release

### PR DESCRIPTION
**Pull request by skriss**
Signed-off-by: Steve Kriss <steve@heptio.com>

### CODE REVIEW

**Comment by ncdc** [comment]
@skriss rebase needed please!

**Comment by skriss** [comment]
done

File: docs/restic.md [hunk 1]
```diff
@@ -123,17 +116,26 @@ You're now ready to use Ark with restic.
...
 ## Restore
 
-Restore from your Ark backup as usual:
+1. Restore from your Ark backup:
 
 ```bash
 ark restore create --from-backup BACKUP_NAME OPTIONS...
 ```
 
+1. When the restore completes, view information about your pod volume restores:
```
[the remark this answers is not in the document]
Comment (skriss) [reply docs/restic.md]: fixed

**Comment by Bradamant3** [comment]
LGTM

**Pull request merged**